comms/core/src/connectivity/manager.rs

```diff
@@ -315,22 +315,18 @@ impl ConnectivityManagerActor {
             },
         }
         match self.pool.get(&node_id) {
-            Some(state) => {
-                if !state.is_connected() {
-                    warn!(
-                        target: LOG_TARGET,
-                        "Existing connection is present but is_connected is false for some reason...."
-                    );
-                }
-
+            Some(state) if state.is_connected() => {
                 if let Some(reply_tx) = reply_tx {
                     let _result = reply_tx.send(Ok(state.connection().cloned().expect("Already checked")));
                 }
             },
-            None => {
+            maybe_state => {
                 info!(
                     target: LOG_TARGET,
-                    "No existing connection found for peer `{}`. Dialing...",
+                    "No connection found for peer (status={}) `{}`. Dialing...",
```

Comment:
```suggestion
                    "No connection found for peer (status={}) `{}`. It might not have existed or did exist but disconnected at some point. Dialling...",
```

Reply:
Used `"Connection is not connected (status={}) `{}`. Dialing...",

Reply:
maybe a comment then?